ner_v1/detectors/textual/name/name_detection.py

```diff
@@ -227,6 +288,176 @@ def context_check_botmessage(botmessage):
+            get_devnagri_text(text=text)
+            >>[u'प्रतिक', u'श्रीदत्त', u'जयराओ']
+        """
+        text = self.replace_stopwords_hindi(text)
+        regex = re.compile(ur"[^\u0900-\u097F\s]+", re.U)
+        text = regex.sub(string=text, repl='')
+        split_text = text.split()
+        if len(split_text) <= 4:
+            return split_text
+
+    def get_hindi_text_from_regex(self, text):
+        """
+        This method is used to detect hindi names using regexes from the given text
+        Args:
+            text (str): text from which hindi names which follow the regex pattern have to be extracted
+
+        Returns:
+            pattern_match (list): list consisting of detected words
+
+        Examples:
+            text = u'मेरा नाम प्रतिक श्रीदत्त जयराओ है'
+            get_hindi_text_from_regex(text=text)
+            >>[u'प्रतिक श्रीदत्त जयराओ']
+
+        """
+        regex_list = [ur"(?:नाम|मैं|हम)\s*([\u0900-\u097F\s]+)",
+                      ur"(?:मुझे|हमें|मुझको|हमको)\s*([\u0900-\u097F\s]+)(?:कहते|बुलाते|बुलाओ)",
+                      ur"\s*([\u0900-\u097F\s]+)(?:मुझे|मैं)(?:कहते|बुलाते|बुलाओ)?"
+                      ]
+
+        for regex in regex_list:
+            regex_ = re.compile(regex, re.U)
+            pattern_match = regex_.findall(text)
+            pattern_match = [self.replace_stopwords_hindi(x) for x in pattern_match if x]
+            if pattern_match:
+                if None != pattern_match[0]:
+                    return pattern_match
+
+    def replace_stopwords_hindi(self, text):
+        """
+        This method is used to replace hindi stop words from the text
+        Args:
+            text (str): The text from which hindi stop words have to be removed
+
+        Returns:
+            clean_text (str): text from which hindi stop words have been removed
+        """
+        split_list = text.split(" ")
+        split_list = [word for word in split_list if word not in HINDI_STOPWORDS]
+        if split_list:
+            return " ".join(split_list)
+
+    def detect_abusive_phrases_hindi(self, text):
+        """
+        This method is used to check for hindi abuses in the sentence
+        Args:
+            text (str): text in which abuses have to be checked
+
+        Returns:
+            status (bool): returns if the text consists of abuses
+        """
+        text = ' ' + text + ' '
+        for abuse in HINDI_BADWORDS:
+            if ' ' + abuse + ' ' in text:
+                return True
+        return False
+
+    def remove_emojis(self, text):
+        """
+        This method is used to remove emojis from the given text
+        Args:
+            text (str): the text from which the emojis have to be removed
+        Returns:
+            text (str): text with emojis replaced with ''
+        """
+        emoji_pattern = re.compile(ur'[{0}]+'.format(''.join(EMOJI_RANGES.values())), re.UNICODE)
+        text = emoji_pattern.sub(repl='', string=text)
```

Comment:
`text = emoji_pattern.sub(replace_with='', replace_string=text)` change to this

Reply:
Didn't get what you're trying to say
_(This is a python function and the arguments are repl, string)_